Change NewSplitDriver paramater and initialization (#76)
open-o11y/opentelemetry-go: CHANGELOG.md

- Modify `BatchSpanProcessor.ForceFlush` to abort after timeout/cancellation. (#1757)
- Improve OTLP/gRPC exporter connection errors. (#1737)
- Make `ExportSpans` in Jaeger Exporter honor context deadline. (#1773)

> - Make `NewSplitDriver` from `go.opentelemetry.io/exporters` take variadic arguments instead of a `SplitConfig` item. 

Reviewer: ```suggestion
- Make `NewSplitDriver` from `go.opentelemetry.io/otel/exporters/otlp` take variadic arguments instead of a `SplitConfig` item. 
```

>   `NewSplitDriver` now automically implements an internal `noopDriver` when all `SplitConfig` fields are not initialized. (#TBD)

Reviewer: ```suggestion
  `NewSplitDriver` now automically implements an internal `noopDriver` for `SplitConfig` fields that are not initialized. (#TBD)
```
